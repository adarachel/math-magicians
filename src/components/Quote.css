@import url("https://fonts.googleapis.com/css2?family=Comic+Neue:wght@700&display=swap");

.loading {
  padding: 2rem;
  font-size: 1.5rem;
  letter-spacing: 0.5rem;
  color: rgb(51, 190, 51);
  font-family: "Comic Neue", cursive;
}

.error {
  padding: 2rem;
  font-size: 1.5rem;
  letter-spacing: 0.2rem;
  font-family: "Comic Neue", cursive;
  color: rgb(193, 37, 37);
}

.quote {
  margin: 1rem;
  padding: 1rem;
  font-family: "Comic Neue", cursive;
}

.quote > h3 {
  font-family: "Comic Neue", cursive;
  text-align: justify;
}

.quote > p {
  font-family: "Comic Neue", cursive;
  text-align: center;
  font-style: italic;
}
