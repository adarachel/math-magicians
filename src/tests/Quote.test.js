import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../components/Quote';

test('Quote component renders correctly', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment()).toMatchSnapshot();
});

test('Quote component displays quote text', () => {
  const { getByText } = render(<Quote />);
  const quoteElement = getByText(/Mathematics is not about numbers/i);
  expect(quoteElement).toBeInTheDocument();
});
