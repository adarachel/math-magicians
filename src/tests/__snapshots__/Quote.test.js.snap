// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="loading"
  >
    Loading
  </div>
</DocumentFragment>
`;
